stubs/Pillow/PIL/ImageFont.pyi

```diff
@@ -74,6 +74,7 @@ class FreeTypeFont:
         stroke_width: float = ...,
         anchor: str | None = ...,
         ink=...,
+        start: tuple[int, int] | None = ...,
```

Comment:
Are you sure these are `int`s? Usually in PIL, coordinates are `float`s. In this case, the values are also passed to `self.font.render()`, though, and I'm not sure what that accepts.

Reply:
Checked the doc on the change: https://pillow.readthedocs.io/en/stable/releasenotes/9.4.0.html#added-start-position-for-getmask-and-getmask2
> fractional coordinates

Sounds like float to me.

It defaulted to an int. But that's probably just because `0` is simpler to write that `0.0`